Musigwa/SendIT-API
# [WIP] #161747510 Fetches the specific user's orders

## What does this PR do (?)

It makes it possible to fetch the specific user's parcel orders only

## Description of Task to be completed (?)

The following should be working:

1. GET /parcels
2. GET /parcels/<parcel_id>
3. GET /users/<userId>/parcels
4. PUT /parcels/<parcelId>/cancel
5. POST /parcels
6. Tests for the all the above endpoints

## How should this be manually tested (?)

After cloning this repo and switch to this branch,

- Using any RESTful Client tool (Usually the postman), test every endpoint
- Make sure to set your Headers as the following:

Key: Content-Type value: application/json

## Any background context you want to provide (?)

I'm using ES6 features in this branch, please be aware of the transipiler (.babelrc file) configurations.

## What are the relevant pivotal tracker stories (?)

1. #161747510
